package.json

```diff
@@ -32,5 +32,8 @@
     "typescript": "^3.7.2",
     "webpack": "^4.41.2",
     "webpack-cli": "^3.3.10"
+  },
+  "bin": {
+    "lpf2-hubinfo": "./bin/hubinfo"
```

Comment:
For consistency with this library name, could we call it `poweredup-hubinfo`? I'm not sure LPF2 is the right name to use given that its Lego internal only, and most people know the technology as Powered UP.